libtransact/src/protocol/key_value_state.rs

```diff
@@ -358,13 +358,11 @@ impl StateEntryList {
     }
 
     pub fn contains(&self, normalized_key: String) -> bool {
-        let nkeys = self
-            .entries()
+        self.entries()
             .to_vec()
             .into_iter()
             .map(|e| e.normalized_key().to_string())
-            .collect::<Vec<String>>();
-        nkeys.contains(&normalized_key)
+            .any(|key| key == normalized_key)
```

Comment:
Probably don’t need the `map` line, either

Reply:
Fixed